### DISCUSSION TITLE
Can't log in to admin page

### DISCUSSION THREAD

**Discussion by ghost** [Keyfactor/ejbca-ce#54] [Q&A]
After bringing up the docker container and enrolling and getting a p12, I add the p12 to Chrome and try to go to the admin page.
After the p12 has been added, I can't connect to the web server anymore. I see this message in the log which may or may not relate to not being able to see the web UI anymore. Has anyone else run into this? Thanks.

2022-06-29 14:54:26,386+0000 INFO  [org.cesecore.certificates.ca.internal.CertificateValidity] (default task-2) Limiting validity of certificate because requested start of validity (Wed Jun 29 14:44:26 UTC 2022) is before CA start of validity (Wed Jun 29 14:52:51 UTC 2022).

**Comment by mike-agrenius-kushner** [comment]
Hi, 

The info message in question is benign, it just tells you that the validity of the P12 was adjusted to not precede the validity of the CA.

So, how is this error manifesting itself? Are you getting an authentication error, or what? =)

Cheers,
Mike

**Comment by ghost** [comment]
After the key has been added, I try to go to the management URL. The
browser cannot connect to the server at all. It is as if the server is not
even running or unreachable.
If I stop the container and start it again, I get prompted again to
enroll/get a key. If you would like, I can set it up and run it again if
there are logs or other information I can provide.

**Reply by mike-agrenius-kushner** [reply]
Yeah, unless you've configured the container to run against a database (in a separate container) you're just running against the built in H2 in Wildfly, which in turn gets wiped when the container closes down. 

But yeah, I'd love to see some logs of what happens on the server when you enroll? Is the Public Web dead as well?
